WalletWasabi/BitcoinCore/Configuration/CoreConfigTranslator.cs

```diff
@@ -18,7 +18,7 @@ public CoreConfigTranslator(CoreConfig config, Network network)
 		public string TryGetValue(string key)
 		{
 			var configDic = Config.ToDictionary();
-			string result = null;
+			string? result = null;
```

Comment:
I would prefer to fix the method as much as possible here. For example, that `result` variable makes no sense and can be removed, you just need to `return found` in the loop and `return null` at the end of the method. After you you that it will be cleat that what the method really returns is a `string?` instead of `string` and in that way we will be able to be alerted about this fact in other parts of the code.

Reply:
> you just need to return found in the loop

I am not sure about this one, because we don't return immediately when `found is { }` but rather we keep iterating.

Reply:
Then use `LastOrDefault`:

```c#
public string? TryGetValue(string key)
{
	var configDic = Config.ToDictionary();
	return NetworkTranslator.GetConfigPrefixesWithDots(Network)
		.Select(networkPrefixWithDot => configDic.TryGet($"{networkPrefixWithDot}{key}"))
		.LastOrDefault(x => x is {});
}
```